src/components/Grid.tsx

```diff
@@ -188,27 +188,26 @@ class Grid extends React.Component<
     switch (activeTab) {
     case "Extensions": {
       const pageOfRepos = await getTaggedRepos("spicetify-extensions", this.requestPage, this.BLACKLIST);
+      const extensions: CardItem[] = [];
       for (const repo of pageOfRepos.items) {
-        const extensions = await fetchExtensionManifest(
+        const repoExtensions = await fetchExtensionManifest(
           repo.contents_url,
           repo.default_branch,
           repo.stargazers_count,
-          this.CONFIG.visual.hideInstalled,
+          true,
         );
 
-        // I believe this stops the requests when switching tabs?
-        if (this.requestQueue.length > 1 && queue !== this.requestQueue[0]) {
-          // Stop this queue from continuing to fetch and append to cards list
-          return -1;
```

Comment:
Do we know for sure what this part did, and that it's okay to take out?

Reply:
Honestly, no clue what it does, I think you added it a while ago to prevent the issue where cards would appear if you switch tabs while it's still loading, however, after experimenting a lot, I don't think this is an issue anymore for whatever reason? (Perhaps changes with how we fetch stuff?)

Reply:
I put it back in, just in case 🤷